src/lib/exec.js

```diff
@@ -7,7 +7,6 @@ async function exec(command, options) {
 	const startTime = Date.now();
 	const exitCode = await _exec(command, null, {
 		...options,
-		silent: true,
```

Comment:
Why is this removed?

Reply:
So that the output of the build/install commands are visible, it makes it a lot easier when smth breaks during build or installation to see what is happening